content/kubermatic/main/tutorials-howtos/networking/cilium-cluster-mesh/_index.en.md

```diff
@@ -0,0 +1,279 @@
++++
+title = "Cilium Cluster Mesh Setup"
+date = 2022-12-07T09:45:00+02:00
+weight = 180
++++
+
+This guide describes the setup for configuring Cilium Cluster Mesh between 2 KKP user clusters
+running with Cilium CNI.
+
+{{< table_of_contents >}}
```

Comment:
I would rather keep this generic. There isn't really that much that would be different on different cloud providers (apart from setting up FW rules etc., but those are very deployment-specific anyway) and it would not be easy to maintain.